alpha_challenges/the_basics.md

```diff
@@ -108,14 +108,14 @@ What is the index value of "Michelangelo"?
 
 **Answer 8A:**
 ```ruby
-# write your code here
+puts jurassic_park['Triceratops']
 ```
 
 * In Ruby, write code to add "Dr. Satler" to the "Humans" of `jurassic_park`.
 
 **Answer 8B:**
 ```ruby
-# write your code here
+puts jurassic_park['Humans'].push('Dr.Satler')
```

Comment:
It might be better to append the value first, then print the output.

**Instead of:**
```
`puts jurassic_park['Humans'].push('Dr.Satler')`
```
**try:**
```
jurassic_park['Humans'].push('Dr.Satler')
puts jurassic_park['Humans']
```

Reply:
Ok, got it! Thanks!